### ISSUE TITLE
Localized account creation

### ISSUE THREAD

**Issue by reneelynn** [SpiderOak/SpiderOakMobileClient#631]
BBQ-896
"When creating a new account in SpiderOak through the app, the information is not localized. This is probably because the request is not sending any l10n information in it.
Please consider this for the future"

**Comment by reneelynn** [comment]
Is this correct or is it more accurately that it's sending us to a web signup? Also correct me if I'm wrong but with changing the way signups work this should go away entirely?

**Comment by kenmanheimer** [comment]
Yes - the app uses a web signup, and the website doesn't have provisions for localizing. There's work starting on localization for the website. The schedule for implementing mobile-app client-signups isn't yet clear, but it could happen sooner, and would be good, too.

**Comment by devgeeks** [comment]
Considering our changes to sign ups, is this still a thing?
